sdk/python/feast/pyspark/launchers/aws/emr_utils.py

```diff
@@ -332,6 +332,7 @@ def _stream_ingestion_step(
             ],
             "Args": ["spark-submit", "--class", "feast.ingestion.IngestionJob"]
             + jars_args
+            + ["--conf", "spark.yarn.isPython=true"]
```

Comment:
Is there any downside of doing this? Do you know why it isn't it always set it to true in Spark?

Reply:
It's yarn specific and from what I found it's used mostly to enable distribution of python related stuff (like pyspark.zip) to yarn workers. It's being set by `spark-submit` when main file is py-file, which is not the case for our IngestionJob.

Reply:
Makes sense. Maybe add a comment there for the future generations?